gulpfile.js

```diff
@@ -318,6 +318,17 @@ gulp.task('connect', ['watch'], function() {
     res.set('Content-Type', 'application/javascript');
     res.send(configJs);
   });
+  //TODO check up with jdyson to figure out how this works on prod
+  hawtio.use('/apiman/config.js', function(req, res, next) {
+    var configJs = 'window.APIMAN_CONFIG = {' +
+      ' auth: {' +
```

Comment:
You tell me :). I want it to participate in the SSO session of the hawtio console.